pkg/apis/stats/v1alpha1/types.go

```diff
@@ -146,3 +146,49 @@ type RuleTrafficStats struct {
 	Name         string       `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
 	TrafficStats TrafficStats `json:"trafficStats,omitempty" protobuf:"bytes,2,opt,name=trafficStats"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +resourceName=nodelatencystats
+// +genclient:onlyVerbs=create,delete,get,list,watch
```

Comment:
I think it doesn't support delete and watch?

Reply:
I asked @IRONICBo to include these verbs. I think we can add their implementation at the same time as the implementation for create, get and list, in the follow-up PR. But maybe we should include the 2 missing stubs in pkg/apiserver/registry/stats/nodelatencystats/rest.go as part of this PR.

Reply:
Yes, I wrote the comment because the server only supports create, get, and list, so had the impression they are the methods we are going to implement. Besides, do we need clients to delete a stats? and I remember supporting standard watch is somewhat difficult due to the resource version stuff.

Reply:
> Besides, do we need clients to delete a stats?

I think we may want Agents to delete their resource if the feature gets disabled. So IMO it makes sense to have delete.

>  I remember supporting standard watch is somewhat difficult due to the resource version stuff.

That makes sense. We should remove watch then.

@IRONICBo do you think you can:
1) remove watch from the list of verbs, and regenerate the code
2) add a stub `Delete` method in the REST server

Reply:
Ok, it sounds good to me.